### ISSUE TITLE
yolov6s reg_output need relu activation？

### ISSUE THREAD

**Issue by dejavvuu** [meituan/YOLOv6#483]
`
            for i in range(self.nl):
                b, _, h, w = x[i].shape
                l = h * w
                x[i] = self.stems[i](x[i])
                cls_x = x[i]
                reg_x = x[i]
                cls_feat = self.cls_convs[i](cls_x)
                cls_output = self.cls_preds[i](cls_feat)
                reg_feat = self.reg_convs[i](reg_x)
                reg_output = self.reg_preds[i](reg_feat)
                
                if self.use_dfl:
                    reg_output = reg_output.reshape([-1, 4, self.reg_max + 1, l]).permute(0, 2, 1, 3)
                    reg_output = self.proj_conv(F.softmax(reg_output, dim=1))
                
                cls_output = torch.sigmoid(cls_output)
                cls_score_list.append(cls_output.reshape([b, self.nc, l]))
                reg_dist_list.append(reg_output.reshape([b, 4, l]))
            
            cls_score_list = torch.cat(cls_score_list, axis=-1).permute(0, 2, 1)
            reg_dist_list = torch.cat(reg_dist_list, axis=-1).permute(0, 2, 1)

            pred_bboxes = dist2bbox(reg_dist_list, anchor_points, box_format='xywh')
            pred_bboxes *= stride_tensor
`

yolov6s没有使用dfl, reg_output应该经过relu, distance ltrb都应该为正数吧

**Comment by Chilicyy** [comment]
@dejavvuu 您好，在训练过程中，当anchor point不在gt里时，distance ltrb有可能为负数，这里不需要经过relu激活函数。

**Comment by dejavvuu** [comment]
@Chilicyy 我看代码里无论atss tal都有gt区域的过滤

**Comment by Chilicyy** [comment]
是的，在训练阶段 模型前向推理之后，label assign会有gt区域过滤，但是上面贴出的代码段是eval评估阶段的预测输出部分，在训练前期，模型学习还不充分时，会有可能出现负数。

**Comment by dejavvuu** [comment]
@Chilicyy 这样子iou计算确认不会有问题吗？而且dfl那边就都保证是正的。感觉没必要出负值

**Comment by Chilicyy** [comment]
@dejavvuu hi, iou计算在label assign之后，所以是没有问题的哈。
